website/content/contributors/getting-started.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Do you want to start contributing to open source and do not know where to start?
```

Comment:
You can see a preview of your article at https://deploy-preview-339--cncf-contribute.netlify.app/contributors/

```suggestion
title: Do you want to start contributing to open source and do not know where to start?
linkTitle: Start Contributing to Open Source
```

This is what the navigation looks like now, and when the `linkTitle` is set, the navigation will show the shorter text from my suggestion. Hopefully it still calls out to the right people to click through and read.

<img width="372" alt="Screenshot 2023-03-10 at 10 06 15 AM" src="https://user-images.githubusercontent.com/1368985/224365087-2d291d2f-3cb2-413a-9cb1-691974d33e85.png">